### PR TITLE
[REVIEW] project file and files GET routes

### LINKED ISSUE #15: Implement GET /users/:userID/projects/:projectID/files/:fileID w/ tests
Returns the contents of a file.

### PR DESCRIPTION
**Pull request by ashleygwilliams**
fixes #15

do we need /users/id/ or can we just use project/id since ids are unique universally, not per user

### CODE REVIEW

**Comment by ashleygwilliams** [comment]
@gideonthomas particularly interested in your thought on this. not including the user lookup makes the query much shorter

**Comment by gideonthomas** [comment]
agree! I think we should do this for the `/files` routes as well and maybe also implement your filter idea

**Comment by ashleygwilliams** [comment]
:+1:

File: api/modules/files/routes.js [hunk 1]
```diff
@@ -13,6 +13,18 @@ exports.register = function(server, options, next) {
...
     }
+  }, {
+    method: 'GET',
+    path: '/projects/{project_id}/files',
+    config: {
+      handler: controller.getProjectFiles
+    }
+  }, {
+    method: 'GET',
+    path: '/projects/{project_id}/files/{id}',
```
Comment (gideonthomas) [inline api/modules/files/routes.js]: `/files/{id}`?

Comment (sedge) [inline api/modules/files/routes.js]: I thought we decided against this in channel?

**Comment by gideonthomas** [comment]
@ashleygwilliams this is good to merge

**Pull request merged**